src/Sentry/iOS/ApiDefinition.cs

```diff
@@ -0,0 +1,1273 @@
+using System;
```

Comment:
is this file code-gen'd?

Reply:
yes, [Objective Sharpie](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/objective-sharpie/)